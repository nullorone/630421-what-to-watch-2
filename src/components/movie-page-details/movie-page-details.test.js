import React from "react";
import renderer from "react-test-renderer";
import MoviePageDetails from "./movie-page-details";

describe(`Make snapshot`, () => {
  const initProps = {};

  it(`Get snapshot component`, () => {
    const tree = renderer
      .create(
          <MoviePageDetails {...initProps}/>
      )
      .toJSON();

    expect(tree).toMatchSnapshot();
  });
});
