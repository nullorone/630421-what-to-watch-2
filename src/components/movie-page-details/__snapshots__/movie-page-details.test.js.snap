// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Make snapshot Get snapshot component 1`] = `
Array [
  <div
    className="visually-hidden"
  >
    <svg
      xmlns="http://www.w3.org/2000/svg"
      xmlnsXlink="http://www.w3.org/1999/xlink"
    >
      <symbol
        id="pause"
        viewBox="0 0 14 21"
      >
        <title>
          Artboard
        </title>
        <desc>
          Created with Sketch.
        </desc>
        <g
          fill="none"
          fillRule="evenodd"
          id="Artboard"
          stroke="none"
          strokeWidth="1"
        >
          <polygon
            fill="#EEE5B5"
            fillRule="nonzero"
            id="Line"
            points="0 -1.11910481e-13 4 -1.11910481e-13 4 21 0 21"
          />
          <polygon
            fill="#EEE5B5"
            fillRule="nonzero"
            id="Line"
            points="10 -1.11910481e-13 14 -1.11910481e-13 14 21 10 21"
          />
        </g>
      </symbol>
    </svg>
  </div>,
  <section
    className="movie-card movie-card--full"
  >
    <div
      className="movie-card__hero"
    >
      <div
        className="movie-card__bg"
      >
        <img
          alt=""
          height="175"
          src=""
          width="280"
        />
      </div>
      <h1
        className="visually-hidden"
      >
        WTW
      </h1>
      <header
        className="page-header movie-card__head"
      >
        <div
          className="logo"
        >
          <a
            className="logo__link"
            href="/"
          >
            <span
              className="logo__letter logo__letter--1"
            >
              W
            </span>
            <span
              className="logo__letter logo__letter--2"
            >
              T
            </span>
            <span
              className="logo__letter logo__letter--3"
            >
              W
            </span>
          </a>
        </div>
        <div
          className="user-block"
        >
          <div
            className="user-block__avatar"
          >
            <img
              alt="User avatar"
              height="63"
              src="../img/avatar.jpg"
              width="63"
            />
          </div>
        </div>
      </header>
      <div
        className="movie-card__wrap"
      >
        <div
          className="movie-card__desc"
        >
          <h2
            className="movie-card__title"
          >
            Keks
          </h2>
          <p
            className="movie-card__meta"
          >
            <span
              className="movie-card__genre"
            >
              
            </span>
            <span
              className="movie-card__year"
            >
              1
            </span>
          </p>
          <div
            className="movie-card__buttons"
          >
            <button
              className="btn btn--play movie-card__button"
              type="button"
            >
              <svg
                height="19"
                viewBox="0 0 19 19"
                width="19"
              >
                <use
                  xlinkHref="#play-s"
                />
              </svg>
              <span>
                Play
              </span>
            </button>
            <button
              className="btn btn--list movie-card__button"
              type="button"
            >
              <svg
                height="19"
                viewBox="0 0 19 19"
                width="19"
              >
                <use
                  xlinkHref="#add"
                />
              </svg>
              <span>
                My list
              </span>
            </button>
            <a
              className="btn movie-card__button"
              href="add-review.html"
            >
              Add review
            </a>
          </div>
        </div>
      </div>
    </div>
    <div
      className="movie-card__wrap movie-card__translate-top"
    >
      <div
        className="movie-card__info"
      >
        <div
          className="movie-card__poster movie-card__poster--big"
        >
          <img
            alt=""
            height="327"
            src=""
            width="218"
          />
        </div>
        <div
          className="movie-card__desc"
        >
          <nav
            className="movie-nav movie-card__nav"
          >
            <ul
              className="movie-nav__list"
            >
              <li
                className="movie-nav__item"
              >
                <a
                  className="movie-nav__link"
                  href="#"
                  onClick={[Function]}
                >
                  Overview
                </a>
              </li>
              <li
                className="movie-nav__item movie-nav__item--active"
              >
                <a
                  className="movie-nav__link"
                  href="#"
                  onClick={[Function]}
                >
                  Details
                </a>
              </li>
              <li
                className="movie-nav__item"
              >
                <a
                  className="movie-nav__link"
                  href="#"
                  onClick={[Function]}
                >
                  Reviews
                </a>
              </li>
            </ul>
          </nav>
          <div
            className="movie-card__text movie-card__row"
          >
            <div
              className="movie-card__text-col"
            >
              <p
                className="movie-card__details-item"
              >
                <strong
                  className="movie-card__details-name"
                >
                  Director
                </strong>
                <span
                  className="movie-card__details-value"
                >
                  Keks
                </span>
              </p>
              <p
                className="movie-card__details-item"
              >
                <strong
                  className="movie-card__details-name"
                >
                  Starring
                </strong>
                <span
                  className="movie-card__details-value"
                >
                  Me
                  , 
                  <br />
                  You
                  , 
                  <br />
                  They
                  , 
                  <br />
                </span>
              </p>
            </div>
            <div
              className="movie-card__text-col"
            >
              <p
                className="movie-card__details-item"
              >
                <strong
                  className="movie-card__details-name"
                >
                  Run Time
                </strong>
                <span
                  className="movie-card__details-value"
                >
                  0h 0m
                </span>
              </p>
              <p
                className="movie-card__details-item"
              >
                <strong
                  className="movie-card__details-name"
                >
                  Genre
                </strong>
                <span
                  className="movie-card__details-value"
                >
                  
                </span>
              </p>
              <p
                className="movie-card__details-item"
              >
                <strong
                  className="movie-card__details-name"
                >
                  Released
                </strong>
                <span
                  className="movie-card__details-value"
                >
                  0h 1m
                </span>
              </p>
            </div>
          </div>
        </div>
      </div>
    </div>
  </section>,
  <div
    className="page-content"
  >
    <section
      className="catalog catalog--like-this"
    >
      <h2
        className="catalog__title"
      >
        More like this
      </h2>
      <div
        className="catalog__movies-list"
      >
        <article
          className="small-movie-card catalog__movies-card"
          onMouseEnter={[Function]}
          onMouseLeave={[Function]}
        >
          <div
            className="small-movie-card__image"
          >
            <img
              alt=""
              height="175"
              src=""
              width="280"
            />
          </div>
          <h3
            className="small-movie-card__title"
          >
            <a
              className="small-movie-card__link"
              href="/films/0"
              onClick={[Function]}
            >
              Keks
            </a>
          </h3>
        </article>
      </div>
    </section>
    <footer
      className="page-footer"
    >
      <div
        className="logo"
      >
        <a
          className="logo__link logo__link--light"
          href="/"
        >
          <span
            className="logo__letter logo__letter--1"
          >
            W
          </span>
          <span
            className="logo__letter logo__letter--2"
          >
            T
          </span>
          <span
            className="logo__letter logo__letter--3"
          >
            W
          </span>
        </a>
      </div>
      <div
        className="copyright"
      >
        <p>
          Â© 2019 What to watch Ltd.
        </p>
      </div>
    </footer>
  </div>,
]
`;
